src/main/java/lottery/Lottery.java

```diff
@@ -0,0 +1,35 @@
+package lottery;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_BALL = 100;
+    private ColorSupplier colorSupplier;
+    private Random randomHelper;
+
+    public Lottery() {
+        colorSupplier = new ColorSupplier();
+        randomHelper = new Random();
+    }
+
+    public ColorSupplier getColorSupplier() {

```

Comment:
please don`t write methods wich is not used

Reply:
Not fixed. Users shouldn't get any information about inner realization, or change realization. Delete all setters and getters for the `Lottery` fields. Your class should contain only the method `getRandomBall()`